lib/reporters/error-reporter.js

```diff
@@ -0,0 +1,37 @@
+const Logger = require('../logger');
+const colors = require('colors');
+const path = require('path');
+const isArray = require('lodash/isArray');
+
+const logger = new Logger();
+
+function errorReporter(errors, logLevel) {
```

Comment:
You're right. I refactored it so that all errors are the treated the same. The errors can still come as a single error or as an array of errors.